### PR TITLE
LPS-188969 Support UPSERT SXPBlueprint and SXPElement

### PR DESCRIPTION
**Pull request by felipelorenz**
https://liferay.atlassian.net/browse/LPS-188969

### CODE REVIEW

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:test:sf

**Comment by felipelorenz** [comment]
ci:test:search

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
ci:reevaluate:1307401_7261

**Review by gustavolimav — CHANGES_REQUESTED** [review]
@felipelorenz Other than those two small changes If tests pass it LGTM.

File: modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPElementResourceImpl.java [hunk 1]
```diff
@@ -73,6 +79,37 @@
 )
 public class SXPElementResourceImpl extends BaseSXPElementResourceImpl {
 
+	public void create(
+			Collection<SXPElement> sxpElements,
+			Map<String, Serializable> parameters)
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPElementResourceImpl.java:84]: Based on workflow we can keep it

https://github.com/felipelorenz/liferay-portal/blob/357f569f2ac814e50cf64a624b830a840748e576/modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/src/main/java/com/liferay/headless/admin/workflow/internal/resource/v1_0/WorkflowDefinitionResourceImpl.java#L72-L73

**Review by gustavolimav — APPROVED** [review]

**Comment by xbrianlee** [comment]
ci:forward:force

**Comment by xbrianlee** [comment]
ci:forward:force

**Pull request closed without merging**